### ISSUE TITLE
CSS conflicts with _bs.scss

### ISSUE THREAD

**Issue by nateplusplus** [kevinchappell/formBuilder#183]
Hey, really small note, but I just thought I would bring it up in case it presents problems with others: there is a style in _bs.scss that was messing with all our styles site-wide and I thought you might want to limit css styles to elements within the formBuilder plugin in order to avoid these conflicts.

The particular style that we had trouble with was: 
```
* {
 -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
}
```
Usually I prefer this type of box-sizing anyway, but there were a few things that got really screwy when it was suddenly applied to every element everywhere.

Perhapse it's better to change this to:
```
.form-wrap.form-builder * {
  -webkit-box-sizing: border-box;
  -moz-box-sizing: border-box;
  box-sizing: border-box;
}
```

**Comment by kevinchappell** [comment]
I tackled this just last week on Formeo and agree it should be changed in formBuilder as well. Will try to push a fix for it today.

**Comment by nateplusplus** [comment]
cool beans, thanks Kevin! There's also some issues with labels... there might be others that need to be confined to formBuilder too.

**Comment by kevinchappell** [comment]
I forgot to put it in the pull request but this should be resolved in #187 . Closing for now but if its still an issue feel free to reopen.

**Comment by willtcarey** [comment]
@kevinchappell The method of encapsulating the styles in _bs.scss looks like it has changed a couple of times but it appears it was removed in the the patch that landed custom controls, seen [here](https://github.com/kevinchappell/formBuilder/commit/edad3b76a32e4cc371f296196d8e9be4f5bb27ba#diff-742ce32fdc6680fbb74be4aac3437e0bL9). 

If that wasn't intentional I'd be happy to submit a pull request to restore that behavior as I'm on a project not using bootstrap and this is screwing with some of styles outside of the form builder.
